pkg/controller/operators/catalog/operator.go

```diff
@@ -323,22 +323,6 @@ func (o *Operator) handleDeletion(obj interface{}) {
 		return
 	}
 
-	// Check the registry server
```

Comment:
Removed this - all we need is the requeue that's done right below, and the main catalog sync takes care of this.